docs/developer-docs/integrations/sns/testing/testing-locally.md

```diff
@@ -30,106 +30,95 @@ These scripts have been most tested with a stack that looks like this:
 
 ## Testing via `sns-testing` repo
 
-A common way to test an SNS locally is to use the `sns-testing` [repo](https://github.com/dfinity/sns-testing#readme).
-
-### Examples of what it tests
-
-The `sns-testing` repo can help with testing the lifecycle of launching an SNS, by running the commands listed [here](https://github.com/dfinity/sns-testing#sns-lifecycle).
-
-For simplicity, you can see the stages of decentralization and how they line up with the CLI commands. 
-
 :::info
 To use the `sns-testing` commands and their arguments properly, please consult the `sns-testing` [README](https://github.com/dfinity/sns-testing#sns-lifecycle).
 :::
 
-### Testing stages of decentralization
 
-Below, we match the stages of decentralization to the commands that the `sns-testing` repo has. Some developers have dapps that do not match the narrow cases of `sns-testing` so the table also includes a column for what other developers have experienced.
+### Testing stages of the SNS launch process
+
+For simplicity, we next map the stages introduced in the [SNS launch stages documentation](../launching/launch-summary-1proposal.md) and in the documentation [Commands & actions to go through SNS launch](../launching/launch-steps-1proposal.md) to the relevant scripts in the `sns-testing` repo so you can learn what part of the launch is tested in which script.
+Note that some developers have dapps that do not match the narrow cases of `sns-testing` so the table also includes a column for what other developers have experienced.
 
 <table border="1">
     <tr>
         <th>Stage Number</th>
         <th>Stage</th>
-        <th>Example `sns-testing` command</th>
+        <th>Example in `sns-testing`</th>
         <th>For other developers</th>
     </tr>
     <tr>
         <td>0</td>
         <td>Developers deploy a dapp to the Internet Computer</td>
-        <td><code>./deploy_test_canister.sh</code></td>
+        <td><code> TODO ./deploy_test_canister.sh</code></td>
         <td>Custom scripts used to deploy dapps (e.g. multi-canister dapps, use nix, etc...)</td>
     </tr>
     <tr>
         <td>1</td>
         <td>Dapp developers choose the initial parameters of the SNS for a dapp</td>
-        <td rowspan="4"><code>./deploy_sns.sh</code></td>
+        <td rowspan="2"><code>TODO ./deploy_sns.sh</code></td>
     </tr>
     <tr>
         <td>2</td>
-        <td>Dapp developers submit NNS proposal so they can deploy to the SNS subnet</td>
+        <td>Submit NNS proposal to create SNS</td>
     </tr>
     <tr>
         <td>3</td>
-        <td>Proposal #1 (of 3) is passed or rejected</td>
+        <td>Dapp developers add NNS root as co-controller of dapp</td>
+        <td><code>TODO</code></td>
+        <td><code></code></td>
     </tr>
```

Comment:
Stages 3 needs to be switched with stage 1 and 2. The nns root has to be in control of the dapps before submitting the proposal

Reply:
Out of curiosity: is this a technical requirement or just a recommendation?
In practice, I think I would recommend to switch 2 and 3 (the choice of the init file should be made and everything should be ready by the time you hand over the SNS to NNS root). WDYT? I'll go ahead and make a proposal in line with this.

Reply:
SGTM.